### PR TITLE
enhancement: Website hosting (#20)

### LINKED ISSUE #20: [Enhancement]: Host GwardiaHub
### My idea

We could use `vercel` to host our website. For database, we could also use vercel or supabase...

### PR DESCRIPTION
**Pull request by Sanielia**
- Migrated database to PostgreSQL,
- Added psycopg2, dj_database_url, whitenoise, gunicorn and uvicorn packages
- Changed SECURITY_KEY into env

### CODE REVIEW

File: render.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - key: DATABASE_URL
+        fromDatabase:
+          name: gwardia_hub
+          property: connectionString
+      - key: WEB_CONCURRENCY
+        value: 4
+      - key: POETRY_VERSION
+        value: 1.8.2
+      - key: DJANGO_DEBUG
+        value: 0
```
Comment (Norbiros) [inline render.yaml:24]:
```suggestion
        value: False
```

**Review by Norbiros — CHANGES_REQUESTED** [review]

**Review by Norbiros — APPROVED** [review]

**Review by tapczanek1321 — APPROVED** [review]

**Comment by SzczurekYT** [comment]
@Kubaryt You changed the way we start the project (we now use docker) without changing the information in readme.
Can you update that too? (as separate prz since this one is merged already)

**Comment by Norbiros** [comment]
We are not using docker. It is planned - https://github.com/Gwardia-Czapli/GwardiaHub/issues/27.

**Comment by SzczurekYT** [comment]
> We are not using docker. It is planned - https://github.com/Gwardia-Czapli/GwardiaHub/issues/27.

Oops I thought render.yaml is a docker compose file.

**Pull request merged**